app/src/main/java/com/codecool/lolapp/activities/MainActivity.kt

```diff
@@ -0,0 +1,58 @@
+package com.codecool.lolapp.activities
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+import com.codecool.lolapp.R
+import com.codecool.lolapp.fragments.DetailsFragment
+import com.codecool.lolapp.fragments.ListingFragment
+import com.codecool.lolapp.fragments.LoreFragment
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+    private var listingFragment: ListingFragment? = null
+    private var detailsFragment: DetailsFragment? = null
```

Comment:
I think it's a bad pattern to create these here. It's dangerous too because of the lifecycle changes they can be null sometimes.  Also, you wouldn't want to keep a reference to something that you might not use later, memory leak garbage collector reasons.